mlflow/utils/proto_json_utils.py

```diff
@@ -485,9 +506,20 @@ def dump_input_data(data, inputs_key="inputs"):
         post_data = {inputs_key: {k: get_jsonable_input(k, v) for k, v in data}}
     elif isinstance(data, np.ndarray):
         post_data = {inputs_key: data.tolist()}
+    elif isinstance(data, list):
+        post_data = {inputs_key: data}
```

Comment:
```suggestion
    elif isinstance(data, (np.ndarray, list)):
        post_data = {inputs_key: list(data)}
```

Reply:
Can't do this because .tolist() of np.ndarry also convert internal data to primitive types.